src/modules/dataGridTable.js

```diff
@@ -0,0 +1,181 @@
+import _ from 'lodash';
+import ClassNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import DataGridTableRow from './dataGridTableRow.js';
+import DragListener from '../utils/dragListener.js';
+import Icon from '../elements/icon';
+import Table from '../collections/table.js';
+
+class DataTable extends React.PureComponent {
```

Comment:
Should we change this class name to `DataGridTable`?

Reply:
fixed.